build.gradle

```diff
@@ -282,7 +281,7 @@ dependencyManagement {
         }
 
         //CVE-2021-22118
-        dependencySet(group: 'org.springframework', version: '5.2.15.RELEASE') {
+        dependencySet(group: 'org.springframework', version: '5.3.9') {
```

Comment:
springframework 5.3.9 has dependancy issues. 5.3.12 is the one that fixed the issues